Hotfix_tests into develop

Setnea opened the pull request:
Adding test6 version working with map_plate changes.

Setnea committed:
Adding test6 conf settings

Test6 still remains some fixes.

Setnea committed:
Adding test6 #107

stork119 commented on tests/release/configuration_settings/2017-02-14-t2.xml:
```diff
@@ -0,0 +1,562 @@
+                                <parameter key="output_path.2" value="wellname_id"/> 
+                            </parameters_by_name>
+                        </TASK>-->
+                        <TASK class="TASK_REMOVE">
```
It should not be commented

stork119 reviewed (changes requested):
Tests have commented parts.

Setnea commented:
Right, I probably was testing sth; temporary commented to speed up process and forgot to uncomment before push. Gonna be more carefull next time.

stork119 reviewed (changes requested):
position.name -> id

Setnea committed:
Fix and update test2 (previously test6) #107

Setnea commented:
Everything works fine now. May I merge?

Setnea committed:
[FIX]

Adding exit() in case of taskname is unknown [to do not waste time on
unproper conf settings]
